languages/kotlin/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,41 @@
+# What's next?
+
+## Submitting your solution
+
+_Hint_: Use `Code --> Reformat` in [IntelliJ IDEA][intellij-idea-ic] to automatically format your code.
```

Comment:
This is great content, but is more suited to be in the [`exercises/shared/.docs/cli.md` file](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#exercisesshareddocsclimd) instead, which is one file that is included for all the exercises.

```suggestion
```

Reply:
Created follow-up issue for this. Thanks.